webhooks/virtualmachine/v1alpha1/mutation/virtualmachine_mutator_unit_test.go

```diff
@@ -348,11 +392,10 @@ func unitTestsMutating() {
 		Context("When VM ImageName is set to a status name matching a single cluster scope image", func() {
 
 			BeforeEach(func() {
-				uniqueStatusName := "unique-status-name"
 				cvmi := builder.DummyClusterVirtualMachineImage("vmi-123")
```

Comment:
nit: `cvmi-123`. ditto on line 405

Reply:
The prefix should still be "vmi-" for cluster images by design.